templates/repo/scripts/Makefile

```diff
@@ -0,0 +1,44 @@
+# Auto-generated by fogg. Do not edit
+# Make improvements in fogg, so that everyone can benefit.
+OS:=$(shell uname -s | tr '[:upper:]' '[:lower:]')
+ifeq ($(shell uname -m),x86_64)
+  ARCH ?= "amd64"
+endif
+ifeq ($(shell uname -m),i686)
+  ARCH ?= "386"
+endif
+ifeq ($(shell uname -m),aarch64)
+  ARCH ?= "arm"
+endif
+
+TF_PRESENT := $(shell command -v terraform 2> /dev/null)
+TF_CURRENT_VERSION := $(shell terraform version | head -n1 | sed 's/Terraform v//g')
+TF_VERSION := 0.11.8
+
+TFLINT_PRESENT := $(shell command -v tflint 2> /dev/null)
```

Comment:
we can do this in a separate pr, but tflint should get installed like a plugin

Reply:
Yeah, still thinking how to reconcile plugins feature with no docker

Reply:
I haven't fully thought this through, but was thinking of managing a `bin` directory in the repo (or one for each arch) and adding that to the `PATH` in the makefiles.

Reply:
yeah, i was thinking something similar. will do in the next pr though so this doesn't get too big